src/containers/AddressPage/styled.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import TextField from '@material-ui/core/TextField';
+
+export const ContainerAddressPage = styled.form `
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+`
+
+export const TextRegister = styled.div `
+    font-size: 16px;
+    color: #000000;
+    height: 18px; 
+    margin-top: 16px;  
+`
+export const InputAddress = styled(TextField) `
+    margin-top: 20px;
+    width: calc(100% - 32px);
+    height: 56px;
+`
+
+export const InputNumber = styled(TextField) `
+    margin-top: 16px;
+    width: calc(100% - 32px);
+    height: 56px;
+`
+
+export const InputHouse = styled(TextField)`
+    margin-top: 16px;
+    width: calc(100% - 32px);
+    height: 56px;
+`
+
+export const InputNeighborhood = styled(TextField)`
+    margin-top: 16px;
+    width: calc(100% - 32px);
+    height: 56px;
+`
+export const InputCity = styled(TextField)`
+    margin-top: 16px;
+    width: calc(100% - 32px);
+    height: 56px;
+`
+
+export const InputState = styled(TextField)`
+    margin-top: 16px;
+    width: calc(100% - 32px);
+    height: 56px;
```

Comment:
Percebam quantas vezes vocês repetiram esse trecho de estilo em específico. Quando esse for o caso vocês podem criar um componente base que tem os mesmo estilos e só extende-lo dependendo do uso de vocês.

Por exemplo:

```jsx
const Button = styled.button`
	margin: 10px;
	padding: 5px;
	color: white;
	background-color: black;
`

// igual ao Button, mas sobrescrevendo a cor e com comportamento de hover
const MyUltraCoolButton = styled(Button)`
	color: green;

	:hover {
		background-color: blue;
	}
`

// igual ao Button, mas com border-radius
const MyVeryElegantButton = styled(Button)`
	border-radius: 15px;
`

```

Percebam, meu `MyUltraCoolButton` tem tudo que o meu componente `Button` tem, a única diferença é a cor e o comportamento no hover. Se algum dia eu quiser mudar alguma característica dos botões eu só preciso alterar o componente `Button`.